app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2><%= I18n.t('auth.sign_in') %></h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
```

Comment:
Same issue before with using `form_with`

Reply:
Fixed in 559bb4a367e391ec4cdfe8eb6c8f929167def53e